.github/workflows/test.yml

```diff
@@ -64,12 +67,16 @@ jobs:
             ONLY_ACTIVE_ARCH=YES
 
   swiftpm-linux:
+    strategy:
+      matrix:
+        swift: ["5.2", "5.6", "5.7"]
```

Comment:
Let's change this to just `5.2` and `5.7` (explicitly so it's clear which exact versions a being used). Otherwise looks great 👍